python-backend/app/api/reporting/resources/metabase.py

```diff
@@ -23,3 +23,32 @@ def get(self):
 
         dashboard_url = current_app.config['METABASE_SITE_URL'] + "/embed/dashboard/" + token.decode("utf8")
         return { 'dashboard_url': dashboard_url }
+
+class LandingPageGraphOneResource(Resource, UserMixin):
+    @requires_role_view_all
+    def get(self):
+        payload = {
+          "resource": {"dashboard": 164},
+          "params": {
+
+          }
+        }
+        token = jwt.encode(payload, current_app.config['METABASE_EMBEDDING_SECRET_KEY'], algorithm="HS256")
+
+        dashboard_url = current_app.config['METABASE_SITE_URL'] + "/embed/dashboard/" + token.decode("utf8")
+        return { 'dashboard_url': dashboard_url }
+
+class LandingPageGraphTwoResource(Resource, UserMixin):
+    @requires_role_view_all
+    def get(self):
+        payload = {
+          "resource": {"dashboard": 165},
+          "params": {
+
+          }
+        }
+        token = jwt.encode(payload, current_app.config['METABASE_EMBEDDING_SECRET_KEY'], algorithm="HS256")
+
+        dashboard_url = current_app.config['METABASE_SITE_URL'] + "/embed/dashboard/" + token.decode("utf8")
```

Comment:
I think we can pull some of this out to avoid the duplication across every resource. Something like:

```python
def get(self):
    dashbord_url = _get_dashboard_url(165)
    return { 'dashboard_url': dashboard_url }


def _get_dashboard_url(self, id):
       payload = {
          "resource": {"dashboard": id},
          "params": {

           }
        }
        token = jwt.encode(payload, current_app.config['METABASE_EMBEDDING_SECRET_KEY'], algorithm="HS256")

        return current_app.config['METABASE_SITE_URL'] + "/embed/dashboard/" + token.decode("utf8")
```